runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -418,9 +418,11 @@ ifeq ($(HOST_ARCH),x)
         SOLINK_SLINK+=dl m
     endif
 
+    ifeq ($(OS),linux)
```

Comment:
Can this check for `!OSX`?  
Please also add a comment that describes why this change was made.  It's much easier to follow the rationale if its in the code than to dig it out of history / issues

Reply:
okay, set to test !OSX